tests/integration/live_data/zookeeper_tests.py

```diff
@@ -48,3 +53,37 @@ def test_create_client_with_credentials(self):
 
         secrets.get_simple.assert_called_with("secret/zk-user")
         self.assertEqual(list(client.auth_data), [("digest", "myzkuser:hunter2")])
+
+    def test_create_client_uses_correct_handler(self):
```

Comment:
🔕  by the existence of the other test class, I think it makes sense to move it to the other one and drop the word "correct" as both are correct depending on the use case. `test_client_uses_gevent_handler_when_queue_patched` helps identify why it should use gevent handler and differentiates it from the other test.

Since the other class has only 1 test in it, you should move the patching to each test so the setUp doesn't impact this test.

Similarly the other class should be named something more like `ZooKeeperHandlerTestsWithPatching`

This is probably one of the few cases that the more "enterprise-y" the name the easier it is for the next person to reason about the test

Reply:
Fair, I suppose the `tearDown` won't affect the other cases so there's no need to be separate.

I've gone with the name `ZooKeeperHandlerWithPatchingTests` to follow the `.*Tests` pattern.